rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -1788,17 +1788,16 @@ bool ppu_load_exec(const ppu_exec_object& elf)
 		mem_size = 0xC800000;
 	}
 
-	if (g_cfg.core.debug_console_mode)
+	if (g_ps3_process_info.get_cellos_appname() == "vsh.self"sv)
```

Comment:
This can be put as first check of the upper if else section then

Reply:
Done.